### PR TITLE
Fix missing podman-remote build options

### PR DESCRIPTION
**Pull request by rhatdan**
Fix handling of SecurityOpts
   LabelOpts
   SeccompProfilePath
   ApparmorProfile

Fix Ulimits

Fixes: https://github.com/containers/podman/issues/9869

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by edsantiago** [comment]
`--target` is referenced in only one buildah test: https://github.com/containers/buildah/blob/f2ef995e8a42d6609ae25ecf90c17281798f39b9/tests/bud.bats#L1525-L1537

**Review by vrothberg — COMMENTED** [review]
LGTM but note that none of the compat-related code seems to be executed in tests at the moment. Do we have something planned to exercise the compat build endpoint?

**Comment by rhatdan** [comment]
@edsantiago -t == --target and that is handled via the Tag option.

+               Tag                    []string `schema:"t"`

**Comment by rhatdan** [comment]
--target=stageName
       Set  the  target build stage to build.  When building a Containerfile with multiple build stages, --target can be used to specify an intermediate build stage by name as the final stage for the resulting image. Commands after the target stage will be skipped.

File: pkg/api/handlers/compat/images_build.go [hunk 1]
```diff
@@ -320,18 +391,22 @@ func BuildImage(w http.ResponseWriter, r *http.Request) {
...
+			CPUPeriod:          query.CpuPeriod,
+			CPUQuota:           query.CpuQuota,
+			CPUShares:          query.CpuShares,
+			CPUSetCPUs:         query.CpuSetCpus,
+			DNSServers:         dnsservers,
+			DNSOptions:         dnsoptions,
+			DNSSearch:          dnssearch,
+			HTTPProxy:          query.HTTPProxy,
+			LabelOpts:          labelOpts,
+			SeccompProfilePath: seccomp,
```
Comment (TomSweeneyRedHat) [inline pkg/api/handlers/compat/images_build.go:405]: To make my OCD, happy, can you move this down a bit in alpha order, please.

**Comment by mheon** [comment]
Restarting bindings test - I think the failure might be a flake?

**Comment by edsantiago** [comment]
Bindings flake is (was) #9902

**Comment by TomSweeneyRedHat** [comment]
@rhatdan can you rebase and see if 9902 fixes this?

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL
This is ready to merge.

**Pull request merged**